app/elements/client-config/client-config.html

```diff
@@ -52,11 +54,14 @@
           config.transforms = {};
           config.transforms.phone = phoneTransform;
           config.transforms.seller = sellerTransform;
-          config.transforms.offer = offerTransform.offer;
+          config.transforms.offer = offerTransform;
           config.transforms.relatedEntities = relatedEntityTransform;
           config.transforms.email = emailTransform;
           config.transforms.webpage = webpageTransform;
           config.transforms.person = personTransform;
+          config.transforms.common = commonTransforms;
+          config.transforms.provider = providerTransform;
```

Comment:
Please add providerTransform to the list of globals at the start of the ready function.